docs/content_management/data_migration/importing_data.md

```diff
@@ -109,6 +109,88 @@ The expression syntax uses the following structure: `###<IDENTIFIER> <EXPRESSION
 
 The `IDENTIFIER` can be any repeated string that encloses the actual expression.
 
+#### Build-in functions
+
+Build-in expression language functions that are tagged with `ibexa.migrations.template.expression_language.function`:
+
+- `ibexa.migrations.template.to_bool` - converts various data types, such as integers or strings, into boolean values
+
+```yaml
+                -   fieldDefIdentifier: show_children
+                    languageCode: eng-US
+                    value: '###XXX to_bool(i % 3) XXX###'
+```
+
+- `ibexa.migrations.template.to_int` - converts different data types, such as strings or floating-point numbers, into integer values
+
+```yaml
+                -   fieldDefIdentifier: quantity
+                    languageCode: eng-US
+                    value: '###XXX to_int("42") XXX###'
+```
+
+- `ibexa.migrations.template.to_float` - transforms various input types, like strings or integers, into floating-point numbers
+
+```yaml
+                -   fieldDefIdentifier: price
+                    languageCode: eng-US
+                    value: '###XXX to_float("19.99") XXX###'
+```
+
+- `ibexa.migrations.template.to_string` - converts values of various types, including integers or floats, into string representations
+
+```yaml
+                -   fieldDefIdentifier: description
+                    languageCode: eng-US
+                    value: '###XXX to_string(123) XXX###'
+```
+
+- `ibexa.migrations.template.reference` - references a specific object or resource within your application or configuration
```

Comment:
This will be a little hard to explain, but...

We're talking about **migration references** here. There are certain things (identifiers, remote IDs, etc.) that can be stored in memory/file during execution of a migration for use in another migration (step or whole file). We could cross-reference the section about migration references here.

Reply:
https://doc.ibexa.co/en/latest/content_management/data_migration/managing_migrations/#references